### ISSUE TITLE
Feedback form error messages should indicate which fields are in error

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/exhibits#2469]
### Current behavior

Feedback form error messages should indicate which fields are in error.

### Steps to reproduce

Example page: [Martin Wong Catalogue Raisonné](https://exhibits.stanford.edu/martin-wong), or any other exhibit.

Use a screen reader (such as VoiceOver) to navigate the Feedback form:
- attempting to submit an incomplete form redirects users to a new page (https://exhibits.stanford.edu/martin-wong/contact), where the Email field is visually indicated as "invalid" but otherwise no alert is given to the screen reader.
- attempting to submit a malformed email string displays a tooltip-style validation error ("Please include an @ in the email address") that is not read by the screen reader.

### Possible solution

WCAG Level A: [Understanding Success Criterion 3.3.1: Error Identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html).

See SODA's suggestions below.

**Note**: This issue is not specific to Spotlight. It also appears in [Arclight Beta](https://archives.stanford.edu/), [Virtual Tribunals](https://virtualtribunals.stanford.edu/), and possibly other applications.

### Reported by SODA
> Violation: Provide a clear indication of fields in error for information that is submitted
> Severity: Major
>
> [Issue]
> There are error messages that do not clearly indicate which fields are in error. On the contact us form in the error state, the error message is not explicitly an error message (doesn't say "error") and is not programmatically associated with the input field. 
> 
> [User Impact]
> When errors are not clearly indicated, users of assistive technology will not know what the error refers to or may not be aware of the existence of errors.
> 
> [Code Reference]
> 
>
> 
> ```
> <div>
>    <label for="contact_form_email">Your email</label>
>    <div>
>       <input type="email" value="" name="contact_form[email]" id="contact_form_email">
>       <div>is invalid</div>
>    </div>
> </div>
> ```
> 
> 
> [Recommendation]
> Use aria-describedby to programmatically associate the input with the error. Also, be sure and include the word Error or equivalent. 
> 
> [Compliant Code Example]
> ```
> <div>
>    <label for="contact_form_email">Your email</label>
>    <div>
>       <input type="email" value="" name="contact_form[email]" id="contact_form_email" aria-describedby="emailError">
>       <div id="emailError">Error: Email is invalid</div>
>    </div>
> </div>
> 
> ```
> [Recommended Reading]
> More information about creating accessible error messages can be found on the Level Access, Deque, and W3C's Websites:
> 
> https://www.levelaccess.com/how-to-provide-accessible-error-identification
> 
> https://www.deque.com/blog/anatomy-of-accessible-forms-error-messages/
> 
> https://www.w3.org/WAI/tutorials/forms/notifications/
